cmd/dex/serve.go

```diff
@@ -469,7 +469,7 @@ func runServe(options serveOptions) error {
 		}
 
 		grpcSrv := grpc.NewServer(grpcOptions...)
-		api.RegisterDexServer(grpcSrv, server.NewAPI(serverConfig.Storage, logger, version))
+		api.RegisterDexServer(grpcSrv, server.NewAPI(serverConfig, logger, version))
```

Comment:
We're just passing this in to get the mock `time.Now`?

Reply:
Also the issuerUrl, I can pass this as a new config, but was thinking it could be useful down the line

Reply:
Ah, I didn't notice that. I'm always sensitive to changes that show up in a few places because it makes maintaining the fork harder, but this is fine.